<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegUser</class>
 <widget class="QMainWindow" name="RegUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1399</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 238, 228);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>1171</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Students</string>
     </attribute>
     <widget class="QListWidget" name="StudentListWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1171</width>
        <height>651</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(182, 255, 151);</string>
      </property>
      <item>
       <property name="text">
        <string>Students</string>
       </property>
       <property name="background">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>161</red>
          <green>35</green>
          <blue>71</blue>
         </color>
        </brush>
       </property>
       <property name="foreground">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Attendance</string>
     </attribute>
     <widget class="QComboBox" name="RoleCombobox">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>0</y>
        <width>368</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(100, 170, 127);
color: rgb(0, 0, 0);</string>
      </property>
      <item>
       <property name="text">
        <string>Assigned Role</string>
       </property>
      </item>
     </widget>
     <widget class="QTableWidget" name="AttendancetableWidget">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>31</y>
        <width>1161</width>
        <height>581</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(182, 255, 151);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>Schedule</string>
     </attribute>
     <widget class="QTableWidget" name="ScheduleWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>1071</width>
        <height>531</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>15</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1200</width>
        <height>1200</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(182, 255, 151);</string>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>570</y>
        <width>551</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addtimeslotbutton">
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(68, 199, 190, 255), stop:1 rgba(255, 255, 255, 255));</string>
         </property>
         <property name="text">
          <string>Add time slot</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removetimeslotbutton">
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(68, 199, 190, 255), stop:1 rgba(255, 255, 255, 255));</string>
         </property>
         <property name="text">
          <string>remove time slot</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Exam</string>
     </attribute>
     <widget class="QTableWidget" name="ExamTableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>1</y>
        <width>1171</width>
        <height>661</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(182, 255, 151);</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="LogOutbutton">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>630</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(102, 44, 188);</string>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>1161</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="CoursesCombobox">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);
color: rgb(0, 0, 0);</string>
       </property>
       <item>
        <property name="text">
         <string>Assigned Courses</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="GroupCombobox">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(137, 185, 101);</string>
       </property>
       <item>
        <property name="text">
         <string>Group</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1399</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
